City-of-Helsinki/drupal-helfi-platform-config
# UHF-8236: Default content

# [UHF-8236](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8236)


## What was done


* Added custom test content importer as running the default content importer service is not an option during a module installation.

## How to install (test on KYMP)
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Switch to correct branch on KYMP and update the Helfi Platform config
    * `git checkout UHF-5924_custom_test_content`  (and pull it)  
    * `composer require drupal/helfi_platform_config:dev-UHF-8236_default_content`
* Run `make drush-cr`

## How to test


* [ ] Run `make shell`, and in shell enable the helfi_test_content module by running `drush en -y helfi_test_content`
* [ ] Check that the `helfi_custom_test_content` module is also installed by running `drush pml | grep test_content`
* [ ] Go to the site and check that you have test content for KYMP custom layouts and paragraphs: district and project layouts, district listing, list of plans and district & project search paragrahs.


[UHF-8236]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8236?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
